gitlab-pipeline/stage/post.yml

```diff
@@ -43,3 +43,40 @@ coveralls:finish-build:
   script:
     - 'curl -k ${COVERALLS_WEBHOOK_URL}?repo_token=${MENDER_COVERALLS_TOKEN} -d "payload[build_num]=$(git rev-parse HEAD)&payload[status]=done"'
 
+.trigger_template:
+  stage: .post
```

Comment:
I think we should have these on a dedicated stage between `publish` and `release`

For other pipelines have triggers only in `.post` makes sense, but for `mender-qa` it would be best if this happens earlier. In a later iteration we would for example not publish anything (iow trigger `release` stage jobs) if `mender-convert` build failed, for example.

Reply:
now moved to a `trigger` stage 👌